test/components/views/settings/__snapshots__/FontScalingPanel-test.tsx.snap

```diff
@@ -1,313 +1,424 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`FontScalingPanel renders the font scaling UI 1`] = `
-<FontScalingPanel>
-  <div
-    className="mx_SettingsTab_section mx_FontScalingPanel"
-  >
-    <span
-      className="mx_SettingsTab_subheading"
-    >
-      Font size
-    </span>
-    <EventTilePreview
-      avatarUrl={null}
-      className="mx_FontScalingPanel_preview"
-      displayName={null}
-      layout="group"
-      message="Hey you. You're the best!"
-      userId={null}
-    >
+{
+  "asFragment": [Function],
```

Comment:
Snapshots need updating to be passed `asFragment()` instead of the whole wrapper